br/tests/lightning_fail_fast_on_nonretry_err/run.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -eu
+
+run_sql 'DROP DATABASE IF EXISTS csv'
+
+out_file_name="$TEST_DIR/sql_res.$TEST_NAME.txt"
+export GO_FAILPOINTS="github.com/pingcap/tidb/br/pkg/lightning/backend/local/WriteToTiKVNotEnoughDiskSpace=return(true)"
+start_time=$(date +'%s')
+run_lightning --backend local > $out_file_name 2>&1 || true
+export GO_FAILPOINTS=""
+check_contains "The available disk of TiKV"
```

Comment:
If `check_contains` fails, it should exit immediately with an error code

Reply:
`set -eu` has been set at the beginning, so it will exit immediately if check_contains fails.

Reply:
OK.  Then I understand why appending a `|| true` at the `run_lightning` command